src/classes/frPledge.cls

```diff
@@ -58,8 +58,9 @@ public class frPledge {
             Pledge_Donation__c = opp.Id,
             Pledge_Donation_uq__c = opp.Id
         );
+
         try {
-            Database.upsert(pledge, Pledge__c.Pledge_Donation_uq__c , true);
+            Database.upsert(pledge, Pledge__c.Fields.Pledge_Donation_uq__c, false);
```

Comment:
Does this matter if it's true or false? Do you want to change this back to false to match the opportunity one?

Reply:
Great catch, amended the original commit to not change this